variables.tf

```diff
@@ -610,6 +614,7 @@ variable "repositories" {
     module-theme-optimization = {
       description = "bfcache, speculative loading, and view transitions for Magento"
       teams       = ["distribution"]
+      users       = ["rhoerr"]
```

Comment:
If you're a tech leader, this is not needed, right?

Reply:
Strictly true, but if/when that status changes in the future, this would keep me as maintainer of that module. I don't need it though.